internal/app/singularity/instance_linux.go

```diff
@@ -23,17 +23,38 @@ type instanceInfo struct {
 	Pid      int    `json:"pid"`
 	Image    string `json:"img"`
 	IP       string `json:"ip"`
+	LogErr   string `json:"logErr"`
+	LogOut   string `json:"logOut"`
 }
 
 // PrintInstanceList fetches instance list, applying name and
 // user filters, and prints it in a regular or a JSON format (if
-// formatJSON is true) to the passed writer.
-func PrintInstanceList(w io.Writer, name, user string, formatJSON bool) error {
+// formatJSON is true) to the passed writer. Additionally, fetches
+// log paths (if showLogs is true).
+func PrintInstanceList(w io.Writer, name, user string, formatJSON bool, showLogs bool) error {
+	if formatJSON && showLogs {
+		sylog.Fatalf("more than one flags have been set")
+	}
+
 	ii, err := instance.List(user, name, instance.SingSubDir)
 	if err != nil {
 		return fmt.Errorf("could not retrieve instance list: %v", err)
 	}
 
+	if showLogs {
+		_, err := fmt.Fprintf(w, "%-16s %-8s %s\n", "INSTANCE NAME", "PID", "LOGS")
+		if err != nil {
+			return fmt.Errorf("could not write list header: %v", err)
+		}
+		for _, i := range ii {
+			_, err := fmt.Fprintf(w, "%-16s %-8d %s\n%-16s %-8s %s\n", i.Name, i.Pid, i.LogErr, "", "", i.LogOut)
```

Comment:
```suggestion
			_, err := fmt.Fprintf(w, "%-16s %-8d %s\n%-26s %s\n", i.Name, i.Pid, i.LogErr, "", i.LogOut)
```
I would suggest combining the white space padding under one formatting directive so that you don't need to pass two empty string args. 

It may be worth looking at a package like [this](https://golang.org/pkg/text/tabwriter). It gives you tab alignment that could come in handy here. You could define a formatting string like `"%s\t%d\t%s\n\t\t%s\n"` preventing the need to pass an empty string at all.

Reply:
I checked out the go TabWriter package and it works great.

Currently, the `singularity instance list` uses fixed alignment (%-16s) and when there exists `len(instanceName) > 16`, the name bleeds into the next column and the alignment is off. However, with TabWriter, the width of each column is dynamically adjusted so the bug goes away. 

Should I also change `singularity instance list` to use TabWriter, or would it be better to log an Issue and make a separate PR to take care of that?

Reply:
Since changing that formatting will be small and will make the formatting in `PrintInstanceList()` uniform for both options, I think it's fine to go ahead and do that in this PR.

Reply:
Just a note thatn if we are changing forwarding for an existing instance list it needs to be noted in the CHANGELOG in case anyone is scraping that output.